qemu/tests/cfg/blockdev_snapshot_multi_disks.cfg

```diff
@@ -44,3 +48,20 @@
         enable_ceph_data2 =yes
         image_format_data1 = raw
         image_format_data2 = raw
+    nbd:
+        enable_nbd_sn1 = no
+        enable_nbd_sn2 = no
+        enable_nbd_data1 = yes
+        enable_nbd_data2 = yes
+        image_format_data1 = raw
+        image_format_data2 = raw
+        nbd_port_data1 = 10831
+        nbd_port_data2 = 10832
+        image_size_data1 = 2G
+        image_size_data2 = 2G
+        image_size_sn1 = ${image_size_data1}
+        image_size_sn2 = ${image_size_data2}
```

Comment:
Can we update line8&15 with the image size, then we can remove these 4 lines

Reply:
After discussion with zhenchao, we decided to keep the original design.